smartmin/csv_imports/models.py

```diff
@@ -8,8 +10,16 @@
 from smartmin.models import SmartModel
 
 
+def generate_file_path(instance, filename):
```

Comment:
What's the reason for this change? Should we be using a random file name instead to ensure uniqueness?